PTCs/CoverageTestingPractice.md

```diff
@@ -0,0 +1,31 @@
+## Test Coverage Progress Tracking
+
+Use this issue to track overall progress towards the long-term goal of *better test coverage*.
+As progress is made check these items off:
+
+- [ ] Functional testing via users.
+      Little or no independent testing.
+- [ ] Independent functional testing of primary capabilities.
+- [ ] Primary functional testing, some unit test coverage.
+- [ ] Comprehensive unit testing, primary functional testing.
+- [ ] Functional testing for documented use cases.
+- [ ] Test coverage commitment.
+
+### Definitions:
+
+- **Functional testing**:
+  Testing capabilities from user's perspective.
+  Many functions can be called.
+  Good for usability assurance.
+  Insufficient to protect against some regressions.
+  Difficult to isolate regressions.
+  Can require extensive execution times.
+- **Unit testing**:
+  Isolated, independent testing of functions and methods.
+  Enable test-driven development, rapid test execution, and fault isolation.
+  Insufficient to ensure functional correctness.
+- **Comprehensive**:
+  Does not mean 100% line coverage, but sufficient coverage to detect most errors.
+  Experts suggest various metrics such as >80% line coverage or a high percentage of function coverage.
+- **Commitment**:
+  Team is committed to writing comprehensive tests concurrent with functionality.
```

Comment:
You'll want to make these bulleted lists under each heading.  GitHub/Lab will treat each of your definitions as one long paragraph otherwise.  You can see how this is rendered [here](https://github.com/betterscientificsoftware/PSIP-Tools/blob/c7263f6c325f8a3ec971e2047f1884d16d3e8ba9/PTCs/CoverageTestingPractice.md).